src/cclib/io/ccio.py

```diff
@@ -363,6 +364,8 @@ def _determine_output_format(outputtype, outputdest):
             outputclass = cmlwriter.CML
         elif outputtype.lower() == 'xyz':
             outputclass = xyzwriter.XYZ
+        elif outputtype.lower() == 'molden':
```

Comment:
https://github.com/cclib/cclib/pull/385/commits/7095913790a7027794a1f8ac658c26d980ab154f I added a unit test for the function.